packages/platform/src/i18n.ts

```diff
@@ -52,6 +52,7 @@ async function loadTranslationsForComponent (component: Component): Promise<Mess
 }
 
 async function getTranslation (message: IntlString): Promise<IntlString | Status> {
+  if (message.indexOf('.') === -1) return message
```

Comment:
This is not correct. A message can contain '.' for example 'Привет.'. We should verify if translation provided and fallback to original `message` if it is not.

Reply:
Just use try/catch when `getTranslation` call? I'm not sure if this is good, perhaps it would be better if `parseId` would not throw an error, but return `undefined`. What's the general concept about errors?

Reply:
Oh got it, so the problem is that `parseId` throws exception... so then your fix makes sense. But let's create `validateId` together with `parseId` and use it instead of `message.indexOf('.') === -1`. `validateId` implementation should match `parseId` so if `parseId` results in exception, then `validateId` returns `false`.